blocks/library/gallery/style.scss

```diff
@@ -19,3 +19,13 @@
 .blocks-gallery__placeholder-instructions {
 	margin: 1.8em 0;
 }
+
+.wp-block-gallery .blocks-editable {
+	position: absolute;
+	bottom: 0;
+	left: 0;
+	right: 0;
+	figcaption {
+		position: static;
```

Comment:
The positioning is a bit messed up because we'd want to apply it to figcaption on the front end, but to editable in the editor.